corehq/apps/app_manager/xpath.py

```diff
@@ -112,6 +113,10 @@ def case(self):
         return CaseXPath(u"instance('casedb')/casedb/case[%s='%s']" % (self.selector, self))
 
 
+class UserCaseXPath(CaseSelectionXPath):
+    selector = 'user_id'
```

Comment:
I think this should be more specific:
```
instance('casedb')/casedb/case[@case_type='user_case'][hq_user_id={user_id in session}]
```
See also: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/callcenter/utils.py#L73-73